.travis.yml

```diff
@@ -1,19 +1,76 @@
 dist: trusty
 sudo: required
-language: node_js
-node_js:
-  - node
-
-install:
+language: android
+android:
+  components:
+    - tools
+    - platform-tools
+    - tools
+    - build-tools-26.0.2
+    - android-26
+    - extra-google-google_play_services
+    - extra-google-m2repository
+    - extra-android-m2repository
+addons:
+  apt:
+    sources:
+      - sourceline: deb https://dl.yarnpkg.com/debian/ stable main
+        key_url: https://dl.yarnpkg.com/debian/pubkey.gpg
+      - sourceline: deb http://dl.google.com/linux/chrome/deb/ stable main
+        key_url: https://dl-ssl.google.com/linux/linux_signing_key.pub
+    packages:
+      - oracle-java8-installer
+      - oracle-java8-set-default
+      - yarn
+      - jq
+      - gradle
+      - google-chrome-stable
+licenses:
+  - android-sdk-preview-license-.+
+  - android-sdk-license-.+
+  - google-gdk-license-.+
+before_cache:
+  - rm -f  $HOME/.gradle/caches/modules-2/modules-2.lock
+  - rm -f  $HOME/.gradle/caches/transforms-1/transforms-1.lock
+  - rm -rf $HOME/.gradle/caches/3.5/fileHashes/
+  - rm -rf $HOME/.gradle/caches/*/plugin-resolution/
+cache:
+  directories:
+    - '$HOME/.gradle/caches/'
+    - '$HOME/.gradle/wrapper/'
+    - '$HOME/.npm'
+before_install:
+  - openssl aes-256-cbc -K $encrypted_b7f1c4c89cb5_key -iv $encrypted_b7f1c4c89cb5_iv
+    -in secrets.tar.enc -out secrets.tar -d
+  - tar xvf secrets.tar
+  - nvm install 8
+  - gem install fastlane
   - npm i -g cordova ionic
-  - npm i
-
 before_script:
   - mkdir www
-  - cp src/assets/data/secret.ts.template src/assets/data/secret.ts
-
+  - cp secret.ts src/assets/data/secret.ts
 script:
+  - npm install
   - npm run lint
   - npm run lint:css
   - cordova prepare
-  - npm run build
+  - cordova plugin ls
+  - cordova platform ls
+  - ionic cordova platform remove android
+  - ionic cordova platform add android@6.4.0
+  - ionic cordova build --release android
+  - jarsigner -verbose -sigalg SHA1withRSA -digestalg SHA1 -storepass $storepass -keystore
+    radar-armt-release-key.keystore $HOME/build/RADAR-base/RADAR-Questionnaire/platforms/android/build/outputs/apk/release/android-release-unsigned.apk
+    alias_name
+  - '/usr/local/android-sdk/build-tools/26.0.2/zipalign -v 4 $HOME/build/RADAR-base/RADAR-Questionnaire/platforms/android/build/outputs/apk/release/android-release-unsigned.apk
+    $HOME/build/RADAR-base/RADAR-Questionnaire/platforms/android/build/outputs/apk/release/radar-armt-app-$APP_VERSION.apk'
+  - if [[ "$TRAVIS_BRANCH" == "master" ]] && [[ "TRAVIS_PULL_REQUEST" == "false" ]]; then
```

Comment:
`$TRAVIS_PULL_REQUEST` missed a `$`

Reply:
You can instead check here if the build is for a tag like -- if [[ "$TRAVIS_BRANCH" == "$TRAVIS_TAG" ]] && [[ "$TRAVIS_TAG" = *alpha* ]] then it will check for only when we release with a tag containing alpha